_docs-v4/event-display/nextDayThreshold.md

```diff
@@ -13,13 +13,72 @@ Only affects timed events that appear on whole-days. Whole-day cells occur in [D
 For example, if `nextDayThreshold` were `'09:00'` (9am), the following event would appear to take up only one day:
 
 ```js
-{ start: '2014-02-04T20:00:00', end: '2014-02-05T02:00:00' }
-// goes from 8pm to 2am the next day
+import { Calendar } from '@fullcalendar/core';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import timeGridPlugin from '@fullcalendar/timegrid';
+
+document.addEventListener('DOMContentLoaded', function() {
+  var calendarEl = document.getElementById('calendar');
+
+  var calendar = new Calendar(calendarEl, {
```

Comment:
this example is really good, but there's a lot to it that don't directly relate to the point you're trying to make. that's why I like to write examples like this...

```js
var calendar = new Calendar(calendarEl, {
  nextDayThreshold: '09:00:00',
  events: [    
    {
      // Goes from 8pm to 2am the next day.
      title: 'Event 1',
      start: '2014-02-04T20:00:00',
      end: '2014-02-05T02:00:00'
    }
  ]
  // other options...
});
```

what do you think?

Reply:
Yeah, agreed. The examples need to be shorter and clutter-free. 

This is the first step. Next, I'll go through the tickets in fullcalendar/fullcalendar#3205 again as well as the docs to see where we can add a bit of clarity or link to this nextDayThreshold or  event-parsing page.